pages/src/main/java/uk/gov/android/ui/pages/modal/BulletedText.kt

```diff
@@ -0,0 +1,130 @@
+package uk.gov.android.ui.pages.modal
+
+import android.content.res.Configuration
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.MaterialTheme.colorScheme
+import androidx.compose.material3.Text
+import androidx.compose.material3.contentColorFor
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import uk.gov.android.ui.theme.m3.GdsTheme
+import uk.gov.android.ui.theme.smallPadding
+
+@Composable
+fun BulletedText(
+    parameters: BulletedTextParameters
+) {
+    with(parameters) {
+        val textColor: Color = colorScheme.contentColorFor(colorScheme.background)
+        val textStyle: TextStyle = MaterialTheme.typography.bodySmall
```

Comment:
Do we want to hard code the text style? Also maybe it can be a private static variable here given it's used in all the composables?

Reply:
yeah fair enough, happy to hard code for now, hopefully we don't need to change text size anytime soon!
I meant just the `textStyle` variable but it's very minor, no need to change 👍